effeciency testing.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+import math
+EffMap = pd.read_csv('analysis/Eff228.csv')
+
+def power_output(map, torque, w, angular_velocity = True):
+    """
+
+    :param map: csv map for torque vs rpm
+    :param torque: torque input
+    :param w: angular velocity input
+    :param angular_velocity: set to false if input for w is in units of RPM instead of angular velocity
+    :return: power output given torque and angular velocity input
+    """
+    torqueprox = round(torque)
```

Comment:
This looks pretty messy tbh - it looks like the torque input is rounded so that the CSV can be directly indexed? Just interpolate between the values instead

Reply:
Its pretty hard to do it any other way tbh